Removed extra phrase "choose the"

HarvestingClouds opened the pull request:
Removed extra phrase "choose the" from section Considerations for scaling threshold values for special metrics

azurecla commented:
Hi __@singhkay__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kasing). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

singhkay commented:
Sent another PR with this change included

HarvestingClouds committed:
Removed extra phrase "choose the" 

Removed extra phrase "choose the" from section Considerations for scaling threshold values for special metrics
